common/lib/xmodule/xmodule/tabs.py

```diff
@@ -758,7 +758,7 @@ def can_display(self, course, settings, *args, **kw):
         the user is one.
         """
         user_is_coach = False
-        if settings.FEATURES.get('CUSTOM_COURSES_EDX', False):
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX', False) and course.enable_ccx:
```

Comment:
Maybe an obvious question: what happens if course.enable_ccx hasn't been set? Most courses in an already-running instance won't have this value set.